adaptive-education-ai/tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+  "compilerOptions": {
```

Comment:
There are a fairly large number of "strict" type checking flags that we should turn on as well